examples/email_report.py

```diff
@@ -34,10 +34,10 @@ def retrieve_api_informations():
     all_cves_filter = {"active": "true"}
     all_cves = client.cve_announcements(all_cves_filter)
 
-    critical_with_exploit_filter = {"exploitable": "true", "active": "true", "level": "level_critical"}
+    critical_with_exploit_filter = {"exploit_code_maturity": "true", "active": "true", "level": "level_critical"}
```

Comment:
Did you test ? Exploit code maturity can't be true.

Reply:
Test for the object with the cve_code 'CVE-2020-4024' returns for the attribute exploit_code_maturity = proof_of_concept. i am modifying the code.